src/rust/engine/options/src/id.rs

```diff
@@ -105,12 +109,8 @@ pub(crate) enum NameTransform {
 }
 
 impl OptionId {
-    pub(crate) fn scope(&self) -> &str {
```

Comment:
It was confusing to have a type called Scope and a function called `scope()` that doesn't return a Scope. Now the field takes over the `scope` name, and its type is Scope.